### ISSUE TITLE
Npm uses older version of argon2_node.cpp, where enum OBJECTS contains THIS instead of THIS_OBJ causing clash with #define THIS void in combaseapi.h (windows default header)

### ISSUE THREAD

**Issue by Stealthmate** [ranisalt/node-argon2#56]
Welcome to the issues section if it's your first time!

**Operating system**: Windows 8.1

**Node version:** 6.2.1

**Compiler version:** 3.10.8

using npm install argon2 , I get an msbuild error 'unexpected 'void' ' at  line 23 in argon2_node.cpp, which is where OBJECTS::THIS is declared. 

I cleaned the npm cache and tried reinstallin argon2, then went into the npm-cache folder, extracted the argon2 tarball and looked through the source - indeed it was THIS. The version in package.js was also 0.13.0, which matches the latest release here on github. 

THIS is defined in combaseapi.h, which is (apparently?) included by default when building and is defined as void, which is the culprit for the error. Interestingly enough, the latest version of argon2_node.cpp here on github seems fixed, because it's "THIS_OBJ" instead of "THIS". 

So in other words, this issue is practically already fixed, but not yet published.

**Comment by ranisalt** [comment]
Yep, I did one more fix prior to publishing and my local npm has broken, now I'm fixing it and will publish 0.14 soon.

**Comment by ranisalt** [comment]
Done!
